demo/full/scripts/controllers/ContentList.tsx

```diff
@@ -434,38 +425,24 @@ function ContentList({
     changeSelectedContent(firstEnabledContentIndex, content);
   }, []);
 
-  const isCustomContent = contentChoiceIndex === 0;
-  const isCustomDRM = chosenDRMType === CUSTOM_DRM_NAME;
-
   const contentsToSelect = contentsPerType[transportType] ?? [];
   const chosenContent = contentsToSelect[contentChoiceIndex];
 
-  let generatedLink = null;
-  if (shouldDisplayGeneratedLink) {
-    generatedLink =
-      contentChoiceIndex === 0 || isSavingOrUpdating
-        ? generateLinkForCustomContent({
-            chosenDRMType: shouldDisplayDRMSettings ? chosenDRMType : undefined,
-            customKeySystem: shouldDisplayDRMSettings ? customKeySystem : undefined,
-            fallbackKeyError: shouldFallbackOnKeyError,
-            fallbackLicenseRequest: shouldFallbackOnLicenseReqError,
-            manifestURL: currentManifestURL,
-            licenseServerUrl: shouldDisplayDRMSettings ? licenseServerUrl : undefined,
-            lowLatency: isLowLatencyChecked,
-            serverCertificateUrl: shouldDisplayDRMSettings
-              ? serverCertificateUrl
-              : undefined,
-            transport: transportType,
-          })
-        : generateLinkForContent(chosenContent, {
-            transportType: transportType,
-            fallbackKeyError: shouldFallbackOnKeyError,
-            fallbackLicenseRequest: shouldFallbackOnLicenseReqError,
-          });
-  }
+  const isCustomContent = React.useMemo(() => {
+    return contentChoiceIndex === 0;
+  }, [contentChoiceIndex]);
```

Comment:
For such a simple operation, I guess there's no difference in using `useMemo`, it just adds complexity no?

Reply:
I don't know, I think if any props of the component is updated, this little calculation would be re-computed if we are not using useMemo. I agree it's not very heavy compute but you think useMemo make this complex ?

Reply:
> if any props of the component is updated, this little calculation would be re-computed if we are not using useMemo

Yes but technically, we're exchanging a simple `contentChoiceIndex === 0` for something like this in React's code:
```js
if (contentChoiceIndex !== previousContentChoiceIndex) {
  return contentChoiceIndex === 0;
} else {
  return memoizedValue;
}
```
So I think using `useMemo` is technically more expensive for nothing here. We're however speaking in both cases of code being extremely fast to compute.

> you think useMemo make this complex?

It's an unnecessary supplementary step here